### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by sabbyanandan**
- remove hardcoded versions
- add bulk app upload step
- correct stream properties
- remove redis requirement
- remove unnecessary patch step

### CODE REVIEW

File: datascience/species-prediction/README.adoc [hunk 1]
```diff
@@ -21,7 +20,7 @@ In order to get started, make sure that you have the following components:
 +
 ```
 $ cd <PATH/TO/SPRING-CLOUD-DATAFLOW>
-$ java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-1.0.0.BUILD-SNAPSHOT.jar
+$ java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-[VERSION].BUILD-SNAPSHOT.jar
```
Comment (markfisher) [inline datascience/species-prediction/README.adoc]: I think `[VERSION]` should immediately precede `.jar`. The "BUILD-SNAPSHOT" should not be present (same applies to several other spots in this PR).

Comment (cppwfs) [inline datascience/species-prediction/README.adoc]: I can fix this on merge.

Comment (ericbottard) [inline datascience/species-prediction/README.adoc]: There is some inconsistency IMHO, where we use `<>` on the `cd` line, and then `[]` to indicate versions. We should choose just one (angle brackets, I believe)

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
Updated [VERSION] with <VERSION>.
Squashed Merged

**Pull request closed without merging**